enterprise_access/apps/api_client/lms_client.py

```diff
@@ -634,13 +634,12 @@ def get_pending_enterprise_group_memberships(self, enterprise_group_uuid):
                             'recent_action': recent_action,
                             'user_email': user_email,
                         })
-            return results
         except requests.exceptions.HTTPError:
             logger.exception('Failed to fetch data from LMS. URL: [%s].', url)
         except KeyError:
             logger.exception('Incorrect data received from LMS. [%s]', url)
 
-        return None
+        return results
```

Comment:
[inform] `results` is initialized as an empty list, so returning `results` here returns an empty list if it wasn't mutated in the `while` iteration above.